iOS/src/toga_iOS/dialogs.py

```diff
@@ -17,7 +17,7 @@ def show(self, host_window, future):
 
         if self.native:
             # Don't differentiate between app and window-modal dialogs.
-            toga.App.app.current_window._impl.native.rootViewController.presentViewController(
+            toga.App.app.current_window._impl.native.rootViewController.presentViewController(  # noqa: E501
```

Comment:
This could be shortened with parentheses, something like:
```python
        if self.native:
            (
                toga.App.app.current_window._impl.native
            ).rootViewController.presentViewController(
                self.native,
                animated=False,
                completion=None,
            )
```